### PR TITLE
Graduate scheduler nodeInclusionPolicy to beta

### PR DESCRIPTION
**Pull request by kerthcet**
Refer to https://github.com/kubernetes/enhancements/issues/3094

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -148,6 +148,7 @@ different Kubernetes components.
 | `MixedProtocolLBService` | `true` | Beta | 1.24 | |
 | `NetworkPolicyStatus` | `false` | Alpha | 1.24 |  |
 | `NodeInclusionPolicyInPodTopologySpread` | `false` | Alpha | 1.25 | |
```
[the remark this answers is not in the document]
Comment (kerthcet) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:150]: Thanks, updated.

**Comment by kerthcet** [comment]
/hold cancel
As related PRs are merged.

**Comment by kerthcet** [comment]
cc @tengqm @sftim for approval.

**Comment by tengqm** [comment]
All upstream code have been merged.
/approve

**Pull request merged**